frontend/src/components/RouteInputContainer.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { IonDatetime, IonDatetimeButton, IonModal } from "@ionic/react";
+
+const RouteInputContainer: React.FC = () => (
+
+    <div>
+        <form style={{ marginBottom: 20 }}>
+            <div>
+                {/* Datetime-Picker basic Ionic-Component */}
```

Comment:
Currently, the user has no way of knowing, what is entered here. There should be a label.